modular_zubbers/code/modules/customization/sprite_accessories/genitals.dm

```diff
@@ -1,2 +1,186 @@
 /datum/sprite_accessory/genital/vagina/cloaca
 	icon = 'modular_zubbers/icons/customization/genitals/vagina.dmi'
+
+/datum/sprite_accessory/genital/penis/human
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Human"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/nondescript
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Nondescript"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/knotted
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Knotted"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/flared
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Flared"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/barbknot
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Barbed, Knotted"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/tapered
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Tapered"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/tentacle
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Tentacled"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/hemi
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Hemi"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/hemiknot
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Knotted Hemi"
+	has_skintone_shading = TRUE
```

Comment:
It might make sense to have a parent type for the sprite accessory like this
```
	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
	name = "Knotted Hemi"
	has_skintone_shading = TRUE
```
and then have all of these be children of it. It would cut down on the amount of lines and would make things easier if the file needed relocated.